### ISSUE TITLE
Node Stabilizer not preventing attraction between nodes

### ISSUE THREAD

**Issue by ThePB** [Azanor/thaumcraft-5#270]
![2016-01-18_16 29 10](https://cloud.githubusercontent.com/assets/5775528/12395297/15015d18-be01-11e5-8025-7010e6ac8a0b.png)
![2016-01-18_16 29 20](https://cloud.githubusercontent.com/assets/5775528/12395295/14ff6620-be01-11e5-985a-5c036c9e7472.png)
![2016-01-18_16 29 28](https://cloud.githubusercontent.com/assets/5775528/12395298/15018180-be01-11e5-8e4f-ca57aa522d7f.png)
![2016-01-18_16 29 34](https://cloud.githubusercontent.com/assets/5775528/12395296/150173e8-be01-11e5-9924-de9211565189.png)

They do not seem to be merging, tho.

Note: The aqua node was originally a little bit lower, when i placed the stabilizer it got pulled up a bit.
The perdito node was originally A LOT higher... something along 10 blocks, and those where pretty much in balance, not pulling each other.

I didn't notice that the perdito node was getting pulled until i already had:
- Closed and reopened minecraft a couple of times
- Loaded and unloaded those chunks a couple of times.

I really have no idea what causes this, but the movment of the perditio node was not seamless... it was pulled in burts, so to speak.

And no, the stabilizer was not the one i tried to break (see the issue about particles), it always remained "on" (excpet for a very brief period when just placed to see if the animation was ok)

TC 5.1.3, baubles 1.1.3.0, forge 1707, journeymap 5.1.3 , Blood Magic 2.0.0 - 3, buildcraft 7.2.0 pre6, chisel and bits 1.8 beta 1, inventory tweaks 1.6 rev 27, jei 2.14.0.67, mantle 0.6 jenkins 60, mmm mmm mmm mmm 1.9b, silent's gems 1.3.8b, TiCo 2.0.1

**Comment by Azanor** [comment]
Tested it several times and I cant duplicate this. Are you sure there wasn't a third node around that the perdito node was being drawn towards? It could have been drawn through the node stabilizer and then got stuck once it was inside the bubble.

**Comment by ThePB** [comment]
There's no underground node in sight. Theaqua node was in fact a little bit underground, it was slowly pulled up by the nodes in the silvertrees. It was a normal node before the first merging.

By the way, the perditio node merged on world reload.

It happens consistently, just had it rehappen with an aqua node. (also merged on world reload) and with another node, spawned for further testing (i tought i was a derp since the silverwood in test one can grow to "cover" the sphere part of the stabilizer, but the second test is clear but still has the issue.

**i might be a derp and forgot that i also have betterfps on. I should really retest this and the totem pole issue on an instance without betterfps before posting.**

**Comment by ThePB** [comment]
Tested with only Thaumcraft 5.1.3... still happening.

![2016-01-19_14 05 05](https://cloud.githubusercontent.com/assets/5775528/12419786/8166c1e8-beb8-11e5-9fa1-eccb3c28c981.png)
![2016-01-19_13 34 26](https://cloud.githubusercontent.com/assets/5775528/12419788/816c915e-beb8-11e5-88b0-80061662bd50.png)
![2016-01-19_13 44 36](https://cloud.githubusercontent.com/assets/5775528/12419789/81759e48-beb8-11e5-8596-440cd17492e4.png)
![2016-01-19_13 51 05](https://cloud.githubusercontent.com/assets/5775528/12419787/81674af0-beb8-11e5-847d-a98825450025.png)

**Comment by Azanor** [comment]
Is this in single player or on a server?

**Comment by ThePB** [comment]
Single player

**Comment by Azanor** [comment]
The fact that it happens very slowly gave me an idea and I figured it out. The node refreshes its state every now and again and rechecks if it is stabilized. That opens up a brief, one-tick window for another node to be drawn towards it. 
Thanks, I'll fix it.

**Comment by ThePB** [comment]
Thanks to you :D

**Comment by MalkContent** [comment]
This is still a thing. I started a silverwood farm and didn't bother with stabilizers for a long time, so a rather large Metallum node grew in the middle of 4 trees. I eventually stabilized it but any new pure nodes from the trees gravitated towards the large node in the middle.
Once arriving in the middle they either got eventually eaten or sometimes pushed the metallum node out of the stabilizer and got stabilized themselves.
It might have something to do with the size of the node being around 100.
@Azanor
